metrics/network/remote_network/README.md

```diff
@@ -4,18 +4,20 @@
 `iperf3`using a remote setup. Host A will run a container that will act as a server while host B will run a
 container that will act as a client. The network bandwidth will be measured across the containers.
 
-- The `remote-networking-memory-smem.sh` script measures RSS memory with `smem` tool. Simultaneously,
+- The `remote-networking-memory-smem.sh` script measures RSS and PSS memory with `smem` tool. Simultaneously,
```

Comment:
"with the `smem` tool" sounds better to me (although strictly that change isn't required by this PR).